src/main/java/pages/ResetPasswordPage.java

```diff
@@ -1,30 +1,25 @@
 package pages;
 
-import base.WebDriverConfig;
+import base.DriverManager;
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebElement;
+import utils.SeleniumHelper;
 
 public class ResetPasswordPage {
-    private By newPass = By.xpath("//input[@id='newPassword']");
-    private By confirmPass = By.xpath("//input[@id='confirmPassword']");
-    private By resetToken = By.xpath("//input[@id='resetToken']");
-    private By resetBtn = By.xpath("//input[@title='Reset password']");
+    private By newPassXPath = By.xpath("//input[@id='newPassword']");
+    private By confirmPassXPath = By.xpath("//input[@id='confirmPassword']");
+    private By resetTokenXPath = By.xpath("//input[@id='resetToken']");
+    private By resetBtnXPath = By.xpath("//input[@title='Reset password']");
 
     public void inputDifferentPasswords(String newPassword, String confirmPassword) {
```

Comment:
rename the function (i.e: `resetPassword`). your function include both enter and click. it's not only input